pyrmq/consumer.py

```diff
@@ -111,6 +111,10 @@ def __init__(
             heartbeat=self.heart_beat,
         )
 
+        self.queue_args["x-queue-type"] = "quorum"
+        if kwargs.get("classic_queue") or "x-max-priority" in self.queue_args:
```

Comment:
don't `or` the priority here. let it raise an error. also, just mention in the readme how to set to classic, no need for a dedicated kwarg for that.